web-registry/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -85,29 +115,48 @@ const Dashboard = (): JSX.Element => {
   );
 
   return (
-    <Box sx={{ bgcolor: 'grey.50' }}>
-      <Section>
-        <IconTabs
-          aria-label="dashboard tabs"
-          tabs={tabs}
-          activeTab={activeTab}
-          linkComponent={Link}
-          mobileFullWidth
-        />
-        <Flex sx={{ ...dashBoardStyles.padTop }}>
-          <Box sx={{ width: '100%' }}>
-            <Outlet
-              context={{
-                isCreditClassCreator,
-                isCreditClassAdmin,
-                isProjectAdmin,
-                isIssuer,
-              }}
-            />
-          </Box>
-        </Flex>
-      </Section>
-    </Box>
+    <>
+      <ProfileHeader
+        name={party?.name ? party?.name : DEFAULT_NAME}
+        backgroundImage={DEFAULT_PROFILE_BG}
+        avatar={party?.image ? party?.image : DEFAULT_PROFILE_AVATAR}
```

Comment:
don't we wanna show a different default avatar in case of organization?  cc/ @erikalogie